src/Symfony/Component/PropertyAccess/Mapping/Cache/DoctrineCache.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Mapping\Cache;
+
+use Doctrine\Common\Cache\Cache;
+use Symfony\Component\PropertyAccess\Mapping\ClassMetadata;
+
+/**
+ * Adapts a Doctrine cache to a CacheInterface.
+ *
+ * @author Luis Ramón López <lrlopez@voutzinos.com>
+ */
+final class DoctrineCache implements CacheInterface
```

Comment:
The new Symfony Cache component (a PSR-6 implementation) has a Doctrine Cache adapter.